### PR TITLE
publication chapter

### LINKED ISSUE #9: Finish publication slides
Workflow:
- [x] pgkdown
- [x] licensing
- [x] opening the GitHub repository
- [x] versioning
- [x] tags on GitHub
- [x] Releases on GitHub
- [x] R-Hub for checks
- [x] CRAN requirements

Exercise:
- [x] running CRAN like checks locally
- [x] Looking at checks on GitHub
- [x] deployment of pkgdown

### PR DESCRIPTION
**Pull request by danielinteractive**
closes #9

### CODE REVIEW

**Comment by danielinteractive** [comment]
@fpahlke would be cool if you could chime in, maybe there is still something important missing - thank you :-)

File: slides/06_publication.qmd [hunk 1]
```diff
@@ -1,8 +1,189 @@
...
+- Typically used via the [website](https://builder.r-hub.io/)
+- There is also an R API: `rhub::check_for_cran()` is comfortable
+:::
+:::
+
+# GitHub
+
+## Tags
+
+- Tags are a feature of Git, i.e. not specific to GitHub
```
Comment (fpahlke) [inline slides/06_publication.qmd]:
```suggestion
- Tags are a feature of Git, i.e., not specific to GitHub
```

File: slides/06_publication.qmd [hunk 2]
```diff
@@ -1,8 +1,189 @@
...
+- Only source packages can be submitted
+<!-- no binary executables are allowed -->
+  - But `.rda` data files are allowed
+- Need single designated maintainer (person, not mailing list)
+  - Additional `Contact` field could be used
+- Citations in author-year style, followed by `<doi:...>`
+- Reducing run time of tests, checks, examples, vignettes is important
+- Need to provide cross-platform portable code: 
+  CRAN runs checks on Windows, Mac, several Linux OS
+
```
Comment (fpahlke) [inline slides/06_publication.qmd:132]:
```suggestion

## CRAN (cont'd)

A CRAN submission can be punishing, painful, and nerve-racking:

The first release of the [rpact](https://cran.r-project.org/package=rpact)
package took 5 weeks and 6 submission attempts; painful experiences:

-   Some Linux machines may generate different random numbers than expected
    (despite setting a seed)
-   Not only errors and warnings lead to the rejection of a submitted package,
    but also hints
-   Your local Windows test system may be much faster than the CRAN system,
    e.g., 5 times
-   Don't use "R Package for" in your package title
-   The description of the package must be provided with a doi reference

## CRAN (cont'd)

Example message informing about the rejection of the last
[rpact](https://cran.r-project.org/package=rpact) submission:

Dear maintainer,

package rpact_3.3.2.tar.gz does not pass the incoming checks automatically,
please see the following pre-tests: \
Windows:
[00check.log](https://cran.r-project.org/web/checks/check_results_rpact.html)
**Status: OK**, Debian:
[00check.log](https://cran.r-project.org/web/checks/check_results_rpact.html)
**Status: OK**

Please fix all problems and resubmit a fixed version via the webform.

Best regards, \
CRAN teams' auto-check service

r-devel-windows-x86_64 Check: Result: NA, Maintainer: 'Friedrich Pahlke'\
r-devel-windows-x86_64 Check: Overall checktime, Result: NOTE, Overall checktime
12 min \> 10 min\
r-devel-linux-x86_64-debian-gcc Check: Result: Note_to_CRAN_maintainers
Maintainer: 'Friedrich Pahlke'

```

**Review by fpahlke — APPROVED** [review]
Looks great 👍 

I only added two slides "CRAN (cont'd)" from my rpact submission experience...

**Comment by danielinteractive** [comment]
@kkmann this would also be more or less ready for merge. maybe you also want to have a quick look first?

**Pull request merged**